features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/policy.js

```diff
@@ -41,7 +41,7 @@ policyModule = function () {
             var queName = "wso2/iot/" + carbonUser.username + "/" + deviceType;
             log.info("Queue : " + queName);
 
```

Comment:
Acknowledged and fixed with commit https://github.com/wso2/carbon-device-mgt/commit/e537e2d841ca3c7946cd34e9da96f261e0ac94e1